components/Typeahead.tsx

```diff
@@ -28,17 +30,26 @@ type Props<T extends TypeaheadModel> = Pick<
  * @returns Typeahead component which can be used to search with any query
  */
 export default function Typeahead<T extends TypeaheadModel>(props: Required<Props<T>>) {
-  const { isLoading, onSearch, renderMenuItemChildren, results, placeholder } = props;
+  const {
+    isLoading,
+    onSearch,
+    renderMenuItemChildren,
+    labelKey,
+    results,
+    placeholder,
+    setSelected,
+  } = props;
   const filterBy = () => true;
 
   return (
     <>
       <AsyncTypeahead
         filterBy={filterBy}
         id="async-typeahead"
-        isLoading={isLoading}
+        isLoading={false}
```

Comment:
Can we add a comment above this to explain why this was done? Might be a bit confusing in the future